parkerbane/MailBux
# ##169287746 Integrate Process And Cluster Management

#### What does this PR do?
Adds PM2 as a process and cluster management tool for the application in staging and production.

#### How should this be manually tested?
Start the application as you would normally and check your [PM2 Dashboard](https://app.pm2.io) online for monitoring.

#### Any background context you want to provide?
You will need to setup an account here [PM2](https://pm2.io) to receive your public/private key pairs which should be added to your `.env`. Follow the `.env.example` guide. The website provides an exhaustive documentation on how to do this.

#### What are the relevant pivotal tracker stories?
[#169287746](https://www.pivotaltracker.com/story/show/169287746)
